### ISSUE TITLE
Catalog search displays duplicate results

### ISSUE THREAD

**Issue by mkenigs** [flox/flox#1427]
`flox search` displays the same result 4 times. I'm also guessing the `10 of 18` results isn't accurate

```
> FLOX_FEATURES_USE_CATALOG=true flox search hello
hello                    A program that produces a familiar, friendly greeting
hello                    A program that produces a familiar, friendly greeting
hello                    A program that produces a familiar, friendly greeting
hello                    A program that produces a familiar, friendly greeting
hello-unfree             An example package with unfree license (for testing)
hello-unfree             An example package with unfree license (for testing)
hello-unfree             An example package with unfree license (for testing)
hello-unfree             An example package with unfree license (for testing)
texlivePackages.othello  Modification of a Go package to create othello boards
texlivePackages.othello  Modification of a Go package to create othello boards

Showing 10 of 18 results. Use `flox search hello --all` to see the full list.

Use 'flox show <package>' to see available versions
```

Expected behavior:
Each search result is unique

**Comment by billlevine** [comment]
This is due to the current implementation of the search index (getting one entry per stability).  
Should be addressed on the service side with [this issue](https://github.com/flox/update-catalog/issues/73)

**Comment by billlevine** [comment]
Resolved with catalog-server#55
